semgrep/tests/e2e/snapshots/test_max_target_bytes/test_max_target_bytes/1.3R/error.txt

```diff
@@ -1,4 +1,4 @@
-Usage: python -m semgrep [OPTIONS] [TARGET]...
-Try 'python -m semgrep --help' for help.
+Usage: python -m semgrep --max-target-bytes [OPTIONS] [TARGET]...
```

Comment:
This is an artifact of adding hidden subcommands to semgrep. A little involved to fix but IMO not worth it since the message is still clear and that command still works :D